Wrangler2 release (#3909)
cloudflare/cloudflare-docs: content/workers/wrangler/migration.md

---
pcx-content-type: how-to
title: Migrating from Wrangler 1
weight: 10
---

## Migration Guide

This document describes the steps to migrate a project from wrangler 1 to 2.

The latest version of wrangler ships with new features and improvements that will require you to include a few changes to your wrangler configuration to get it working as expected.

The good news is that the CLI itself should guide you through the upgrade process.

To learn more about new updates in Wrangler, refer to the [What's new page](/workers/wrangler/compare-v1-v2/).

### Upgrade to Wrangler 2

> The first step is to either install Wrangler globally or use the `npx command` to run it against your Wrangler 1 project. Then, run the `wrangler dev` command to generate a list of deprecation warnings or fields which need to be changed in order to build your Worker using Wrangler 2.

Reviewer: > install Wrangler globally
It might be worth spelling this out for users that already have `@cloudflare/wrangler` installed. If you do, and you try to `npm install -g wrangler` you'll get an error that prompts you to delete the file or use the `--force` option to overwrite

The plan with this new version is to make developing with Cloudflare Workers seamless and improve the overall experience of the developers building on the platform.

> To learn more about what's new in wrangler, refer to the [What's new page](/workers/wrangler/compare-v1-v2/).

Reviewer 2: ```suggestion
To learn more about new updates in Wrangler, refer to the [What's new page](/workers/wrangler/compare-v1-v2/).
```

Reviewer 2: We also do not do contractions in the dev docs. What is the 'what's new' page? Is there a PR for it? This needs to be renamed.

### Deprecations

> The following commands is now deprecated in wrangler 2.0, if you're migrating, be sure to go through the list and remove the commands that is no longer required:

Reviewer 2: ```suggestion
The following commands are deprecated in Wrangler 2.0, if you are migrating, be sure to go through the list and remove the commands that are no longer required:
```

> - Remove the `type` and `webpack_config` property in the config that is no longer required.

Reviewer 2: By "config", do we mean wrangler.toml file here?

Reviewer 3: Yes I meant, the wrangler.toml file.
